python/hooks/check.bash

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env bash
+# shellcheck disable=SC2030,SC2031
```

Comment:
This was because of a bug i shellcheck right? Maybe add some info or link about it